homeassistant/components/ezviz/camera.py

```diff
@@ -147,6 +248,53 @@ def update(self):
     async def async_added_to_hass(self):
         """Subscribe to ffmpeg and add camera to list."""
         self._ffmpeg = self.hass.data[DATA_FFMPEG]
+        entities = self.hass.data.setdefault(EZVIZ_DATA, {}).setdefault(ENTITIES, [])
+        entities.append(self)
+        _LOGGER.debug("Registering services for entity_id=%s", self.entity_id)
+        async_dispatcher_connect(
+            self.hass, f"ezviz_switch_ir_on_{self.entity_id}", self.switch_ir_on
+        )
+        async_dispatcher_connect(
+            self.hass, f"ezviz_switch_ir_off_{self.entity_id}", self.switch_ir_off
+        )
+        async_dispatcher_connect(
+            self.hass, f"ezviz_switch_audio_on_{self.entity_id}", self.switch_audio_on
+        )
+        async_dispatcher_connect(
+            self.hass, f"ezviz_switch_audio_off_{self.entity_id}", self.switch_audio_off
+        )
+        async_dispatcher_connect(
+            self.hass,
+            f"ezviz_switch_privacy_on_{self.entity_id}",
+            self.switch_privacy_on,
+        )
+        async_dispatcher_connect(
+            self.hass,
+            f"ezviz_switch_privacy_off_{self.entity_id}",
+            self.switch_privacy_off,
+        )
+        async_dispatcher_connect(
+            self.hass, f"ezviz_switch_state_on_{self.entity_id}", self.switch_state_on
+        )
+        async_dispatcher_connect(
+            self.hass, f"ezviz_switch_state_off_{self.entity_id}", self.switch_state_off
+        )
+        async_dispatcher_connect(
+            self.hass,
+            f"ezviz_switch_follow_move_on_{self.entity_id}",
+            self.switch_follow_move_on,
+        )
+        async_dispatcher_connect(
+            self.hass,
+            f"ezviz_switch_follow_move_off_{self.entity_id}",
+            self.switch_follow_move_off,
+        )
+
+    async def async_perform_ptz(self, direction, sleep):
```

Comment:
Looks like this should be speed not sleep

Reply:
indeed!